PhoneShop/src/main/java/com/javanaitei/phoneshop/entity/Advertising.java

```diff
@@ -0,0 +1,23 @@
+package com.javanaitei.phoneshop.entity;
+
+public class Advertising {
+	private Integer id;
+	private String image_list;
```

Comment:
KO đặt tên method là `_` nhé

Đặt tên ko bắt đầu với khoảng trasnwgs và ko nên chứa giá trị đặt biệt vì em gen `set/get` sẽ bị sai đó 

đặt theo kiểu `CamelCase` -> `imageList`

cơ mà chỗ ni em tận dụng số ít số nhìu, khỏi thêm từ list vào hì 
```suggestion
	private String images;
```